README.md

```diff
@@ -1 +1,69 @@
-# budget_app
\ No newline at end of file
+# Budget app
+
+Budget app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## Built With
+
+- Ruby -v 3.0.2
+- Rails -v 7.0.4
+- gem 'rspec'
+- gem 'sass-rails'
+
+### Presentation
+
+[Video](https://www.loom.com/share/711cc33bc1644273b2a30b881b96c54d)
+
+### Live 
+
+[budgetapp](https://budget-app-arnaud.herokuapp.com/)
+
+### Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+#### Prerequisites
+
+- Ruby -v 3.0.2
+- Rails -v 7.0.4
+- Postgresql
+
+#### Setup
+
+- Open the terminal and run the following commands:
+
+```
+- git clone git@github.com:ArnaudBand/budget_app.git
+- rails new . --database=postgresql -T
+- cd budget_app
+
+```
+
+- Run the application on the server using this command
+
+`rails s`
+
```

Comment:
- [ ] Kindly also add instructions in your ReadMe on how to run the test cases of your project, under the Getting Started section.